Content.Server/Holopad/HolopadSystem.cs

```diff
@@ -771,6 +784,28 @@ private HashSet<Entity<HolopadComponent>> GetLinkedHolopads(Entity<HolopadCompon
         return linkedHolopads;
     }
 
+    private void UpdateHolopadControlLockoutStartTime(Entity<HolopadComponent> source)
+    {
+        if (!TryComp<TelephoneComponent>(source, out var sourceTelephone))
+            return;
+
+        var sourceTelephoneEntity = new Entity<TelephoneComponent>(source, sourceTelephone);
+
+        var query = AllEntityQuery<HolopadComponent, TelephoneComponent>();
+        while (query.MoveNext(out var receiver, out var receiverHolopad, out var receiverTelephone))
+        {
+            var receiverTelephoneEntity = new Entity<TelephoneComponent>(receiver, receiverTelephone);
+
+            if (!_telephoneSystem.IsSourceInRangeOfReceiver(sourceTelephoneEntity, receiverTelephoneEntity))
+                continue;
+
+            if (receiverHolopad.ControlLockoutStartTime > source.Comp.ControlLockoutStartTime)
+                source.Comp.ControlLockoutStartTime = receiverHolopad.ControlLockoutStartTime;
+        }
+
+        Dirty(source);
+    }
```

Comment:
This never actually checks if the data is dirtied and networks it anyway, is it always expected to be dirty?